test/fuzz/gen.ml

```diff
@@ -121,14 +134,17 @@ let expr_available_2_f32 =
 (* let expr_available_3_f32 = [] *)
 
 let expr_available_1_f64 env =
-  [ pair B.funop_64 (const [ S.Nothing ])
-  ; pair B.i32_trunc_f64 (const [ S.Pop; S.Push (Num_type I32) ])
-  ; pair B.i64_trunc_f64 (const [ S.Pop; S.Push (Num_type I64) ])
-  ; pair B.i32_trunc_sat_f64 (const [ S.Pop; S.Push (Num_type I32) ])
-  ; pair B.i64_trunc_sat_f64 (const [ S.Pop; S.Push (Num_type I64) ])
-  ; pair B.f32_demote_f64 (const [ S.Pop; S.Push (Num_type F32) ])
+  [ pair B.f32_demote_f64 (const [ S.Pop; S.Push (Num_type F32) ])
   ; pair B.i64_reinterpret_f64 (const [ S.Pop; S.Push (Num_type I64) ])
+  ; pair B.funop_64 (const [ S.Nothing ])
   ]
+  @ ( if env.Env.conf = Concrete then
+        [ pair B.i32_trunc_f64 (const [ S.Pop; S.Push (Num_type I32) ])
+        ; pair B.i64_trunc_f64 (const [ S.Pop; S.Push (Num_type I64) ])
+        ; pair B.i32_trunc_sat_f64 (const [ S.Pop; S.Push (Num_type I32) ])
+        ; pair B.i64_trunc_sat_f64 (const [ S.Pop; S.Push (Num_type I64) ])
+        ]
+      else [] )
```

Comment:
Could you put all of them as before and let the fuzzer fail if they're not implemented ? We'll implemented them later.